src/app/modules/document-detail/document-create/document-create.component.spec.ts

```diff
@@ -33,14 +45,18 @@ describe('DocumentCreateComponent', () => {
         HttpClientTestingModule,
         RouterTestingModule,
         ReactiveFormsModule,
+        TranslateModule.forRoot({
```

Comment:
You can and should use the TranslateTestingModule

Reply:
Not using now